config.md

```diff
@@ -304,46 +304,32 @@ For Windows based systems the user structure has the following fields:
 
 ## <a name="configPlatform" />Platform
 
-**`platform`** (object, REQUIRED) specifies the configuration's target platform.
-
-* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) file system bundle.
-  The runtime MUST generate an error if it does not support the specified **`os`**.
-  Bundles SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
-  If an operating system is not included in the `GOOS` documentation, it SHOULD be submitted to this specification for standardization.
-* **`arch`** (string, REQUIRED) specifies the instruction set for which the binaries in the specified [`root`](#root) file system bundle have been compiled.
-  The runtime MUST generate an error if it does not support the specified **`arch`**.
-  Values for **`arch`** SHOULD use, and runtimes SHOULD understand, **`arch`** entries listed in the Go Language document for [`GOARCH`][go-environment].
-  If an architecture is not included in the `GOARCH` documentation, it SHOULD be submitted to this specification for standardization.
+**`platform`** (string, REQUIRED) specifies the configuration's target platform.
+  The value MUST be a slug from [the platform list](spec.md#platforms).
 
 ### Example
 
 ```json
-"platform": {
-    "os": "linux",
-    "arch": "amd64"
-}
+"platform": "linux"
 ```
 
 ## <a name="configPlatformSpecificConfiguration" />Platform-specific configuration
 
-[**`platform.os`**](#platform) is used to specify platform-specific configuration.
+[**`platform`**](#platform) is used to specify platform-specific configuration.
 Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
 
 * **`linux`** (object, OPTIONAL) [Linux-specific configuration](config-linux.md).
-  This MAY be set if **`platform.os`** is `linux` and MUST NOT be set otherwise.
+  This MAY be set if **`platform`** is `linux` and MUST NOT be set otherwise.
 * **`windows`** (object, OPTIONAL) [Windows-specific configuration](config-windows.md).
-  This MAY be set if **`platform.os`** is `windows` and MUST NOT be set otherwise.
```

Comment:
Actually, this MUST be set if the platform is windows. A container isn't going to run at all without it. Not sure whether this applies to other platforms though.